core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1062,8 +1062,9 @@ private void createFileAndJournal(LockedInodePath inodePath, CreateFileOptions o
     mTtlBuckets.insert(inode);
 
     if (options.isPersisted()) {
-      // The path exists in UFS, so it is no longer absent.
-      mUfsAbsentPathCache.process(inodePath.getUri());
+      // The path exists in UFS, so it is no longer absent. The ancestors exist in UFS, but the
+      // actual file does not exist in UFS yet.
+      mUfsAbsentPathCache.processExisting(inodePath.getUri().getParent());
```

Comment:
When will we invalidate the created path? This isn't necessary since we will hit the Alluxio metadata, but would it be safe to invalidate the path?